config.yaml

```diff
@@ -7,3 +7,13 @@ batch_write_size:
 batch_flush_interval:
 # Encoding of stored data. Either json or protobuf. Default json.
 encoding:
+# Whether to use TLS connection. Default false.
+tls_connection:
```

Comment:
why are these changes here again? Didn't we merge this before?

Reply:
We definitely did, I don't know it's marked as a change

Reply:
By looking at the commits the PR includes some that should not be here https://github.com/pavolloffay/jaeger-clickhouse/pull/26/commits. E.g. The "expose metrics"

Reply:
nit: when working on a PR create a branch from the main/head`

```
git pull upstream main
git checkout -b feature-name
```

Then if there is a conflict or you want to synch with the main branch use rebase:
```
git rebase -i upstream main
```

Before running the rebase I would recommend squashing all commits (git rebase -i <hash of the commit>)

Reply:
> By looking at the commits the PR includes some that should not be here https://github.com/pavolloffay/jaeger-clickhouse/pull/26/commits. E.g. The "expose metrics"

It actually happen due to rebasing branch. Maybe I did it incorrectly, gonna try redoing it properly

Reply:
> gonna try redoing it properly

Done